assets/src/blocks/Accordion/AccordionEditorScript.js

```diff
@@ -39,14 +38,15 @@ registerBlockType(BLOCK_NAME, {
   keywords: [
     'accordion',
     'faq',
-    'collapsible'
+    'collapsible',
   ],
   supports: {
     html: false, // Disable "Edit as HTMl" block option.
   },
   attributes,
   edit: AccordionEditor,
-  save: ({ attributes }) => {
+  // eslint-disable-next-line no-shadow
```

Comment:
As less as we add this skips, we'll have a cleaner code.

Reply:
Ok, thanks for the feedback